include/picongpu/particles/synchrotron/AlgorithmSynchrotron.hpp

```diff
@@ -0,0 +1,431 @@
+/* Copyright 2014-2024 Marco Garten, Jakob Trojok, Filip Optolowicz
```

Comment:
I do not think that Marco and Jakob must be mentioned here. Year should be 2024.

Reply:
done